server/src/test/java/io/druid/client/DirectDruidClientTest.java

```diff
@@ -314,7 +313,7 @@ public void testQueryInterruptionExceptionLogMessage() throws JsonProcessingExce
 
     TimeBoundaryQuery query = Druids.newTimeBoundaryQueryBuilder().dataSource("test").build();
     HashMap<String, List> context = Maps.newHashMap();
-    interruptionFuture.set(new ByteArrayInputStream("{\"error\":\"testing\"}".getBytes()));
+    interruptionFuture.set(new ByteArrayInputStream("{\"error\":\"testing1\",\"errorMessage\":\"testing2\"}".getBytes()));
```

Comment:
Actually looking further, the old code doesn't even serialize the exception directly (just the message) so even if you query the historicals/peons directly, you still wouldn't have seen it.